src/_locales/en/messages.json

```diff
@@ -24,7 +24,7 @@
         "description": "Header of popup window"
     },
     "renderingLabel": {
-        "message": "Interpretation",
+        "message": "Rendering",
```

Comment:
Any specific reason to change it to "Rendering"? it was specifically changed in PR: https://github.com/Shared-Reality-Lab/IMAGE-browser/pull/135

Reply:
> @Clarisa1999 , Why only Audio Renderings are set to default? Is this the expected behaviour? I think by default both Audio & text renderings should be selected. Please make the necessary changes.

@Cybernide from what I understand from the wireframe and our meetings, audio is the only one that is default right?

Reply:
<img width="479" alt="Screen Shot 2022-02-09 at 4 33 01 PM" src="https://user-images.githubusercontent.com/55144801/153293831-3c2f5254-42ac-413d-9483-5d05c3c1cb20.png">

Reply:
I think text should be off and audio should be the default